storm-core/test/clj/org/apache/storm/nimbus_test.clj

```diff
@@ -46,7 +46,7 @@
   (:import [org.apache.commons.io FileUtils])
   (:import [org.json.simple JSONValue])
   (:import [org.apache.storm.daemon StormCommon])
-  (:import [org.apache.storm.cluster IStormClusterState StormClusterStateImpl ClusterStateContext ClusterUtils])
+  (:import [org.apache.storm.cluster DaemonType IStormClusterState StormClusterStateImpl ClusterStateContext ClusterUtils])
```

Comment:
Harmless, but don't see tests using `DaemonType` in this file.

Reply:
Will fix it.